.kuzzlerc

```diff
@@ -118,9 +118,11 @@
       "controllers": {
         "auth": {
           "actions": {
-            "login": true,
+            "logout": true,
```

Comment:
I discussed about that with @stafyniaksacha and had at the beginning the same advice as you...
... but a user that has an expired token is automatically mapped to anonymous, and he may want to logout while his token is still expired... In that case, it'd be better to allow the action (that just do nothing) instead of reponse by an UnauthorizedError.